lib/users/api.js

```diff
@@ -713,17 +713,19 @@ class API {
    * @throws {APIError} if the request was unsuccessful
    */
   async isJobExpired(jobId=undefined, job=undefined) {
-    let expirationDate;
     if (jobId) {
       let jobDetails = await this.getJobDetails(jobId);
-      expirationDate = jobDetails.expiration_date;
-    } else if (job) {
-      expirationDate = job.expiration_date;
-    } else {
+      return jobDetails.expired;
```

Comment:
Same as https://github.com/voxel51/api-py/pull/86, this will require further backend changes to support